### PR TITLE
[ci] Only run portion of tests on PR draft

### PR DESCRIPTION
**Pull request by feisuzhu**
Related issue = #5545

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -427,7 +447,7 @@ jobs:
         env:
           PY: py38
           GPU_TEST: ON
-          TI_WANTED_ARCHS: 'cpu,cuda,vulkan,opengl'
```
Comment (qiao-bo) [inline .github/workflows/testing.yml:430]: we don't need cpu here because?

**Review by qiao-bo — APPROVED** [review]
LGTM!

**Comment by qiao-bo** [comment]
btw, how much time has been reduced here by enabling the lite option?

File: .github/workflows/testing.yml (hunk 1 above)
Comment (feisuzhu) [reply .github/workflows/testing.yml:430]: Because CPU are tested in dedicated cpu tasks

**Comment by feisuzhu** [comment]
> btw, how much time has been reduced here by enabling the lite option?

Halved :)

**Pull request merged**